docs/install.rst

```diff
@@ -3,24 +3,27 @@
 Installation
 ============
 
-This part of the documentation covers the installation of the dyn module.
-The first step to using any software package is getting it properly installed.
+This part of the documentation covers the installation of the Dyn module.
 
 
 Distribute & Pip
 ----------------
 
-Installing dyn is simple via `pip <http://www.pip-installer.org/>`_.
+The easiest way to install the Dyn module is via ‘pip’.
+
+1. Go to <http://www.pip-installer.org/>’, download and install ‘pip’.
```

Comment:
I totally understand that and questioned whether to put this in or not, but I decided to drop it in because I tend to look at most things as "baby's first [insert whatever technology]" when I document them. Making it as accessible as possible is for the best, I think.